views/components/BarGraphTooltip/helpers.js

```diff
@@ -1,8 +1,13 @@
-import { N_A, TOTAL_LABEL } from '../../../constants'
+import { N_A, TOTAL_LABEL, TOTALS } from '../../../constants'
 
-const formatTooltipData = (studyCounts = 0, studyTargets) =>
+export const formatTooltipData = (studyCounts = 0, studyTargets) =>
   !!studyTargets ? `${studyCounts} / ${studyTargets}` : `${studyCounts}`
 
+export const formatTotalsTooltipData = (
+  { siteCount, siteTarget },
```

Comment:
out of curiosity, why are these in an object and the other parameter is not?

Reply:
haha good catch.
https://github.com/AMP-SCZ/dpdash/pull/266/commits/9ef8e4026d6002baf27d9287e747ff5d083b7f50